CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# dbt_intercom v0.5.0
+## 🚨 Breaking Changes 🚨
+- In this PR ([#25](https://github.com/fivetran/dbt_intercom/pull/25)), we've added:
```

Comment:
These changes were actually implemented in [dbt_intercom_source PR#18](https://github.com/fivetran/dbt_intercom_source/pull/18). Can we add that distinction.

Reply:
@fivetran-joemarkiewicz Whoops, fixed now.